mozilla/blurts-server
# Submit unlocalised strings, hide "Exposure type" outside the US

# References: 
Jira: #3969
Figma: N/A




# Description

This primarily hides the "Exposure type" column on exposure cards when the user can only ever see one exposure type (data breaches). This should be safe because that column is already hidden on mobile, so the CSS should already be prepared for this - but cc @codemist to be sure.

I also moved to strings that are visible for non-US users from `dashboard-premium.ftl` to `dashboard.ftl`.

# Screenshot (if applicable)

Non-US:

![image](https://github.com/mozilla/blurts-server/assets/4251/93591b2f-da13-4ac2-8efd-a19015c40380)

US:

![image](https://github.com/mozilla/blurts-server/assets/4251/4b4748d5-45c2-4b2e-b18e-5029a92747bf)


# How to test

Visit the dashboard with a non-US user - verify that the column doesn't show up.

Visit the dashboard with a US user (even if they don't have Premium) - the column should show up.

# Checklist (Definition of Done)
